scripts/ldmx-env.sh

```diff
@@ -7,9 +7,24 @@
 #   2. Can run docker as a non-root user
 ###############################################################################
 
-#this is the name of the dockerhub repository
-#   it shouldn't change
-export _docker_hub_repo="ldmx/dev"
+if [ -z $3 ]
+then
+    _repo_name="dev"
+    #this is the name of the dockerhub repository
+    #   it shouldn't change
+    # except it should when we want a production container
+    # also, if it's set to "local", skip downloading at all and attempt to use a locally built container
+else
+    _repo_name="$3"
+fi
+export _docker_hub_repo="ldmx/$_repo_name"
+#compatibility tweak. we could also suggest that all local builds are named with prefix convention ldmx/local: instead of ldmx/dev:
+if [ ${_repo_name} = "local" ]
+then
+    export _docker_hub_repo="ldmx/dev"
+fi
+echo $_docker_hub_repo
```

Comment:
Could you remove this line?

Reply:
It just looks like a line you used in debugging and would just clutter the output. If you think it is important to show the user the docker hub repo, I think thats cool. Then I would ask for you to fill out this line with a short sentence about what the script has decided.

Reply:
sorry i forgot to push! now a few things should be prettier, like this intermediate check line being gone

Reply:
Thank you!